### PR TITLE
AspectList: Connect with holder bear

### PR DESCRIPTION
**Pull request by adhikasp**
Replacement of https://github.com/coala/coala/pull/4111
Closes https://github.com/coala/coala/issues/4094

### CODE REVIEW

**Comment by adhikasp** [comment]
Accidentally push enter :(

Anyway, I was trying to keep the original @pratyushprakash commit but there is many rebase and logic conflict so I modify them.

File: tests/bears/BearTest.py [hunk 1]
```diff
@@ -111,6 +112,10 @@ def run(self, y: int, w: float):
         yield w
 
 
+class languagesTestBear(Bear, languages=['Python', 'C#']):
```
Comment (sils) [inline tests/bears/BearTest.py]: why lower case?

Comment (adhikasp) [inline tests/bears/BearTest.py]: I believe it's a mistake (and I miss it)

**Comment by adhikasp** [comment]
@jayvdb @userzimmermann

**Comment by userzimmermann** [comment]
ack https://github.com/coala/coala/pull/4490/commits/97226cad17e58819117df756a9d92e24e2d6ad36 https://github.com/coala/coala/pull/4490/commits/b7fabc12c87ec089eb024e642a6f63b802c72d7e

**Comment by userzimmermann** [comment]
@rultor merge

**Comment by jayvdb** [comment]
unack 9557ab7

**Comment by jayvdb** [comment]
unack'd due to https://github.com/coala/coala/issues/4094#issuecomment-316879556

File: tests/bears/BearTest.py [hunk 2]
```diff
@@ -183,6 +194,10 @@ def test_detect_and_fix_aspects(self):
         assert (CommitMessage.Shortlog.TrailingPeriod in
                 aspectsTestBear.aspects['fix'])
 
+    def test_aspectlist_bear(self):
+        assert aspectsTestBear.aspects['fix'].bear is aspectsTestBear
```
Comment (jayvdb) [inline tests/bears/BearTest.py]: use `assertIs`

File: tests/bears/BearTest.py [hunk 3]
```diff
@@ -144,6 +149,12 @@ def tearDown(self):
 
 class BearTest(BearTestBase):
 
+    def test_languages(self):
+        assert type(LanguagesTestBear.languages) is Languages
```
Comment (jayvdb) [inline tests/bears/BearTest.py]: use `assertIn`

**Comment by jayvdb** [comment]
ack 579b417 d036e92

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**